tests/test_e2e_90_kafka_events.py

```diff
@@ -0,0 +1,118 @@
+import os
+import time
+import json
+import asyncio
+
+import pytest
+import requests
+from aiokafka import AIOKafkaConsumer
+from aiokafka.admin import AIOKafkaAdminClient, NewTopic
+
+from tests.helpers import NetworkTest
+
+CONTROLLER = '127.0.0.1'
+KYTOS_API = 'http://%s:8181/api/kytos' % CONTROLLER
+
+class TestE2EKafkaEvents:
+    net = None
+
+    def setup_method(self, method):
+        """
+        It is called at the beginning of every class method execution
+        """
+        # Start the controller setting an environment in
+        # which all elements are disabled in a clean setting
+        self.net.config_all_links_up()
+        self.net.start_controller(clean_config=True, enable_all=False)
+        self.net.wait_switches_connect()
+        time.sleep(5)
+
+    @classmethod
+    def setup_class(cls):
+        cls.net = NetworkTest(CONTROLLER)
+        cls.net.start()
+        cls.net.wait_switches_connect()
+
+    @classmethod
+    def teardown_class(cls):
+        cls.net.stop()
+
+    @pytest.mark.asyncio
+    async def test_napp_starts_correctly(self):
```

Comment:
We usually have a `test_<\d+>_<\w+>...` number in the test case name that we use for other test suites, so it'd be great to have something similar. They are picked arbitrarily monotonically for each test suite, so they end up great to help with quick test selection too with their name prefixes